lib/message.rb

```diff
@@ -0,0 +1,35 @@
+require 'twilio-ruby'
+# require 'dotenv'
```

Comment:
Think you were very much on the right lines with this. The 'staff' answer also uses this dotenv approach to access environmental variables (rather than needed to have them hard-coded into this file). Just needed an extra couple of steps to make it work, I guess...?